ext_emconf.php

```diff
@@ -1,31 +1,31 @@
 <?php
 
 $EM_CONF[$_EXTKEY] = [
-    'title' => 'typo3_forum',
-    'description' => 'Forum extension',
-    'category' => 'plugin',
-    'author' => 'Mittwald CM Service',
-    'author_email' => 'support@mittwald.de',
-    'author_company' => 'Mittwald CM Service',
-    'dependencies' => 'cms,extbase,fluid,sr_feuser_register,static_info_tables',
-    'state' => 'stable',
-    'uploadfolder' => 0,
-    'createDirs' => 'typo3temp/typo3_forum,typo3temp/typo3_forum/gravatar',
-    'modify_tables' => 'fe_users',
-    'clearCacheOnLoad' => 0,
+    'title'            => 'typo3_forum',
+    'description'      => 'Forum extension',
+    'category'         => 'plugin',
+    'author'           => 'Mittwald CM Service',
+    'author_email'     => 'support@mittwald.de',
+    'author_company'   => 'Mittwald CM Service',
+    'dependencies'     => 'cms,extbase,fluid,sr_feuser_register,static_info_tables',
```

Comment:
Key `dependencies` is deprecated and should not be used anymore (see [TYPO3 CoreApiReference](https://docs.typo3.org/typo3cms/CoreApiReference/8.7/ExtensionArchitecture/DeclarationFile/) for v8).  
I suggest we drop this line.

Reply:
has been removed